src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -79,6 +79,8 @@ const _MatCheckboxBase = mixinColor(
     '[attr.tabindex]': 'null',
     '[class._mat-animation-noopable]': `_animationMode === 'NoopAnimations'`,
     '[class.mdc-checkbox--disabled]': 'disabled',
+    '[class.mdc-checkbox-indeterminate]': 'indeterminate',
+    '[class.mdc-checkbox-checked]': 'checked',
```

Comment:
I don't see these classes defined anywhere in MDC's CSS.

Reply:
Yes these should be `mat-mdc-...`

Reply:
I don't think that we even need to set them since they aren't used for anything.